numpy/core/tests/test_multiarray.py

```diff
@@ -5918,6 +5921,10 @@ def aligned(n):
             itemsize=aligned(size + 1)
         ), (3,)))
 
+        expected_dtype = {'names': ['f0'], 'formats': ['i'],
+                          'itemsize': np.dtype('i,V1', align=True).itemsize}
+        self._check('(3)T{ix}', (expected_dtype, (3,)))
```

Comment:
Although perhaps using `np.dtype(..., align=True)` is an improvement over `aligned(size + 1)`